.github/workflows/UnitTests.yml

```diff
@@ -102,7 +102,11 @@ jobs:
       run: |
         docker run -v /home/runner/actions-runner/_work/maxtext/maxtext:/app --rm --privileged maxtext_base_image bash -c \
         'bash end_to_end/test_generate_param_only_checkpoint.sh -r runner_$(date +%Y-%m-%d-%H-%M) -o gs://runner-maxtext-logs -d gs://maxtext-dataset -i 4'
-  
+    - name: Test gamma
+      run: |
+        docker run -v /home/runner/actions-runner/_work/maxtext/maxtext:/app --rm --privileged maxtext_base_image bash -c \
```

Comment:
this really should be XLML! Because it is slow. If we want to do it here, can we restrict it be just the 2.5B model?

Reply:
(For the TPUs, we run this in XLML already. I'm not sure on GPUs for XLML, could be a good chat with @RissyRan )

Reply:
Yes, currently MaxText tests are running TPU. If you are working on GPU, We have it available in XL ML via 2 ways: 1) GKE via [XPK](https://github.com/google/xpk) AND 2) GCE directly. The GCE route currently only support single slice but we are working on multislice feature.

Here attaches some examples for your reference to onboard Gamma into XL ML:
* GKE example on TPU: https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/examples/xpk_example_dag.py#L34-L54. I think @michelle-yooh has landed GKE with GPU.
* GCE example on GPU: https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/pytorch_xla/pytorchxla_torchbench.py#L37-L46

Happy to chat more!

Reply:
Ok, deleted test_gamma from UnitTests.yml. 
@michelle-yooh  Could you confirm if you have already added GKE with GPU for maxtext end_to_end?
If so, I will do it using GCE directly. @RissyRan Do I need to do code change after multislice feature is enabled?